CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Not released
 
+- Formula widget supports custom aggregation expression [699](https://github.com/CartoDB/carto-react/pull/699)
+- Fix executeModel get url calculation [699](https://github.com/CartoDB/carto-react/pull/699)
```

Comment:
fix... affecting what part of widgets? all?

Reply:
yes, it's for all remote-mode & global widgets